models/demos/mamba/tt/mamba_one_step_ssm.py

```diff
@@ -24,6 +26,8 @@ def __init__(self, args: ModelArgs, device, configs, load_fn: Callable):
         self.configs = configs
         self.n = 32
         self.rank = self.args.dt_rank
+        self.seq_len = args.seq_len
```

Comment:
Do we need to store this? I think we could just infer sequence length from the input in `forward`, that way we can reducing the amount of state we are keeping.

If we only support certain sequence lengths we can just assert it in `forward`

Reply:
I guess maybe we need to store it because of the `return x.repeat(self.batch_size * self.seq_len, 1)` on the A parameter...

Should we just fix the seq_len member variable to 32  for now and don't actually pass it as a arg?

Reply:
I think we should take care of seq_len here itself as we most likely have to do prefill for seq_len>32 in the functional implementation. Getting error on tt conv for seq_len=32